x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/json_tab.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { CodeEditor } from '@kbn/kibana-react-plugin/public';
+import { XJsonLang } from '@kbn/monaco';
+import { CspFinding } from '../types';
+
+const offsetHeight = 120;
+
+export const JsonTab = ({ data }: { data: CspFinding }) => (
+  <div style={{ position: 'absolute', inset: 0, top: offsetHeight }}>
```

Comment:
couldn't figure out why but there is some offset to the editor when using the `inset: 0` method. I'm using this constant to set it in place. its a random but I don't think it should create any problems for us. let me know if you have any ideas how so solve this otherwise